packages/common/src/FormValidationError.ts

```diff
@@ -0,0 +1,11 @@
+type FormValidationErrors = Record<string, string[]>;
+
+export class FormValidationError extends Error {
+  public errors: FormValidationErrors;
+
+  constructor(errors: FormValidationErrors) {
+    super(Object.values(errors)[0][0]);
```

Comment:
Would it be better to do a big csv join of all errors here? Otherwise either something dynamic or static. I worry about the edge case where there's an empty `errors` passed in and the indexing `Object.values(errors)[0][0]` null refs.

Reply:
Yes, good point. We removed the `string[]` type of the errors object in a different PR, but currently, this could fail indeed.

https://github.com/Videodock/ott-web-app/pull/86/files#diff-9caec313adc28576ac57907ce90dd950c36466ed49cce2f3d8179edb284e5b18